lib/sequelize.js

```diff
@@ -952,7 +953,7 @@ class Sequelize {
    * const cls = require('continuation-local-storage');
    * const ns = cls.createNamespace('....');
    * const Sequelize = require('sequelize');
-   * Sequelize.cls = ns;
+   * Sequelize.useCls(ns);
```

Comment:
It would be possible to keep the old API, but `Sequelize.cls` would have to become a setter, so I figured better to make it explicit.

But happy to go with what you think is best.